spec/syntax.md

```diff
@@ -227,19 +227,34 @@ Any such future keyword must start with `.`,
 followed by two or more lower-case ASCII characters.
 
 The rest of the statement supports
-a similarly wide range of content as _reserved annotations_,
-but it MUST end with one or more _expressions_.
+a similarly wide range of content as _reserved annotations_ (with the exception
+of U+002E FULL STOP `.`, which is only allowed following characters that are
+permissible in _name_), but it MUST end with one or more _expressions_.
 
 ```abnf
-reserved-statement = reserved-keyword [s reserved-body] 1*([s] expression)
+reserved-statement = reserved-keyword [s reserved-statement-body] 1*([s] expression)
```

Comment:
With the change proposed here, could we not also drop the requirement for one or more expressions? If a `.` preceded by whitespace or `}` is no longer allowed in a reserved statement, then we can allow for e.g. `.strict` just by itself to be a valid `reserved-statement`.

CC @gibson042 about this.

```suggestion
reserved-statement = reserved-keyword [s reserved-statement-body] *([s] expression)
```

Reply:
I would also be OK with this.

Reply:
Breadcrumb for when I get to this: #547